linkedin/server.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
-import BaseHTTPServer
-import urlparse
+from http.server import BaseHTTPRequestHandler, HTTPServer
+import time
+from urllib.parse import urlparse

```

Comment:
I got an error using this import with python 3 because in the handler parameter it uses the method `parse_qs` that's not recognized from `urlparse`.

I used instead: `from urllib import parse`

And then in the params: `params = parse.parse_qs(p[1], True, True)`

I'm still having other issues doing my request, so although I'm pointing this out to you, I recommend a double check on that. Thank you, hope that was of some help. 

[https://docs.python.org/3/library/urllib.parse.html](https://docs.python.org/3/library/urllib.parse.html)